data-raw/test_dataset.R

```diff
@@ -103,14 +105,14 @@ dbWriteTable(con_testdb, "RegSpecies_3eSet", regspecies_3eset, overwrite = TRUE)
 
 vegetation <-
   data.frame(
-    IDPlots = c(rep(20, 3), rep(60, 2)), ID = 1:5,
-    Total_moss_cover = c(NA, 15, 20, 10, 10),
-    Total_herb_cover = c(NA, 15, 20, 10, 10),
-    Total_shrub_cover = c(NA, 15, 20, 10, 10),
-    Total_tree_cover = c(NA, 15, 20, 10, 10),
-    Total_waterlayer_cover = c(NA, 15, 20, 10, 10),
-    Total_SoildisturbanceGame = c(NA, 15, 20, 10, 10),
-    Date = c(rep(NA, 2), 1436233600, 1022716800, 1122716800)
+    IDPlots = c(rep(19, 2), rep(20, 3), rep(60, 2)), ID = c(1,2, 1:5),
```

Comment:
```suggestion
    IDPlots = c(rep(19, 2), rep(20, 3), rep(60, 2)), ID = c(1, 2, 1:5),
```
lintr gaat hierop reclameren...